### DISCUSSION TITLE
How can I implement a custom syntax?

### DISCUSSION THREAD

**Discussion by RaenonX** [remarkjs/remark#770] [General] [answered]
Say if I want to enlarge `text` like this `!!text!!`, how can I implement that?

I am currently using `react-markdown`. I tried using `component` options for doing such, but I found that nested syntax like `!!**Bolded and enlarged**!!` won't work.

**Comment by wooorm** [comment, marked as answer]
See https://github.com/micromark/micromark#extensions

**Reply by wooorm** [reply]
That being said: you probably **should not**, as outlined in that chapter
